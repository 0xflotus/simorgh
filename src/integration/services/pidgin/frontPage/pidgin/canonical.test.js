/**
 * @service pidgin
 * @pathname /pidgin
 */

import runCanonicalTests from '../../../../pages/frontPage/canonicalTests';

describe('Canonical pidgin frontPage', runCanonicalTests);
