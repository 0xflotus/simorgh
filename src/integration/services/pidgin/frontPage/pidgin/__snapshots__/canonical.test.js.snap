// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Canonical pidgin frontPage A11y I can see the skip to content link 1`] = `"Waka go wetin de inside"`;

exports[`Canonical pidgin frontPage Analytics ATI 1`] = `"<img height=\\"1px\\" width=\\"1px\\" alt=\\"\\" style=\\"position:absolute\\" src=\\"https://logws1363.ati-host.net?s=598343&amp;s2=70&amp;p=pidgin.page&amp;x1=[urn:bbc:cps:52f3c7b1-83ed-2338-e050-17ac8045cd94]&amp;x2=[responsive]&amp;x3=[news-pidgin]&amp;x4=[pcm]&amp;x7=[index-home]&amp;x8=[simorgh-nojs]&amp;x9=[Domot+-+BBC+News+Pidgin]&amp;x11=[1970-01-18T08:30:40.608Z]&amp;x12=[1970-01-19T01:24:09.011Z]\\"/>"`;

exports[`Canonical pidgin frontPage Footer I can click on the BBC branding and it would take me to the homepage 1`] = `"/pidgin"`;

exports[`Canonical pidgin frontPage Footer I can see the footer copyright and external linking text 1`] = `"Â© 2020 BBC. De external site no concern BBC. De way wey we de take go external link."`;

exports[`Canonical pidgin frontPage Header I can see the branding 1`] = `"BBC News, Pidgin"`;

exports[`Canonical pidgin frontPage I can see Index Alsos 1`] = `"Another thing we de for inside dis tori"`;

exports[`Canonical pidgin frontPage I can see sections 1`] = `"Top tori"`;

exports[`Canonical pidgin frontPage I can see sections 2`] = `"Yarn Me Tori"`;

exports[`Canonical pidgin frontPage I can see sections 3`] = `"Sport (click for more tori) See everitin"`;

exports[`Canonical pidgin frontPage I can see sections 4`] = `"Ginger me"`;

exports[`Canonical pidgin frontPage I can see sections 5`] = `"Informate me"`;

exports[`Canonical pidgin frontPage I can see sections 6`] = `"Special reports 2"`;

exports[`Canonical pidgin frontPage I can see sections 7`] = `"2018 World Cup"`;

exports[`Canonical pidgin frontPage I can see sections 8`] = `"Podcast"`;

exports[`Canonical pidgin frontPage I can see the headline 1`] = `"BBC News, Pidgin - Home"`;

exports[`Canonical pidgin frontPage SEO Canonical link 1`] = `"http://localhost:7080/pidgin"`;

exports[`Canonical pidgin frontPage SEO Dir attribute 1`] = `"ltr"`;

exports[`Canonical pidgin frontPage SEO FB admins 1`] = `"100004154058350"`;

exports[`Canonical pidgin frontPage SEO FB app ID 1`] = `"1609039196070050"`;

exports[`Canonical pidgin frontPage SEO Heading level 1 1`] = `"BBC News, Pidgin - Home"`;

exports[`Canonical pidgin frontPage SEO Lang attribute 1`] = `"pcm"`;

exports[`Canonical pidgin frontPage SEO Linked data 1`] = `
Object {
  "@context": "http://schema.org",
  "@type": "WebPage",
  "image": Object {
    "@type": "ImageObject",
    "height": 576,
    "url": "https://news.files.bbci.co.uk/ws/img/logos/og/pidgin.png",
    "width": 1024,
  },
  "mainEntityOfPage": Object {
    "@id": "http://localhost:7080/pidgin",
    "@type": "WebPage",
    "name": "Domot",
  },
  "publisher": Object {
    "@type": "NewsMediaOrganization",
    "logo": Object {
      "@type": "ImageObject",
      "height": 576,
      "url": "https://news.files.bbci.co.uk/ws/img/logos/og/pidgin.png",
      "width": 1024,
    },
    "name": "BBC News Pidgin",
    "publishingPrinciples": "https://www.bbc.com/pidgin/institutional-48528766",
  },
  "thumbnailUrl": "https://news.files.bbci.co.uk/ws/img/logos/og/pidgin.png",
  "url": "http://localhost:7080/pidgin",
}
`;

exports[`Canonical pidgin frontPage SEO OG description 1`] = `"We dey give una latest tori on top politics, environment, business, sports, entertainment, health, fashion and all di oda things wey dey happen for West and Central Africa come add di rest of di world join. For better informate plus explanation of all di ogbonge tori wey pipo never hear about for inside West and Central Africa, BBC Pidgin dey serve am with video, audio, maps and oda graphics join."`;

exports[`Canonical pidgin frontPage SEO OG image 1`] = `"https://news.files.bbci.co.uk/ws/img/logos/og/pidgin.png"`;

exports[`Canonical pidgin frontPage SEO OG image alt 1`] = `"BBC News Pidgin"`;

exports[`Canonical pidgin frontPage SEO OG locale 1`] = `"pcm"`;

exports[`Canonical pidgin frontPage SEO OG site name 1`] = `"BBC News Pidgin"`;

exports[`Canonical pidgin frontPage SEO OG title 1`] = `"Domot - BBC News Pidgin"`;

exports[`Canonical pidgin frontPage SEO OG type 1`] = `"website"`;

exports[`Canonical pidgin frontPage SEO OG url 1`] = `"http://localhost:7080/pidgin"`;

exports[`Canonical pidgin frontPage SEO Page title 1`] = `"Domot - BBC News Pidgin"`;

exports[`Canonical pidgin frontPage SEO Robots meta tag 1`] = `"noodp,noydir"`;

exports[`Canonical pidgin frontPage SEO Twitter card 1`] = `"summary_large_image"`;

exports[`Canonical pidgin frontPage SEO Twitter creator 1`] = `"@BBCNews"`;

exports[`Canonical pidgin frontPage SEO Twitter description 1`] = `"We dey give una latest tori on top politics, environment, business, sports, entertainment, health, fashion and all di oda things wey dey happen for West and Central Africa come add di rest of di world join. For better informate plus explanation of all di ogbonge tori wey pipo never hear about for inside West and Central Africa, BBC Pidgin dey serve am with video, audio, maps and oda graphics join."`;

exports[`Canonical pidgin frontPage SEO Twitter image alt 1`] = `"BBC News Pidgin"`;

exports[`Canonical pidgin frontPage SEO Twitter image src 1`] = `"https://news.files.bbci.co.uk/ws/img/logos/og/pidgin.png"`;

exports[`Canonical pidgin frontPage SEO Twitter site 1`] = `"@BBCNews"`;

exports[`Canonical pidgin frontPage SEO Twitter title 1`] = `"Domot - BBC News Pidgin"`;

exports[`Canonical pidgin frontPage Story Promo I can see a headline 1`] = `"AS E DE HAPPEN Senators don begin vote for Senate President"`;

exports[`Canonical pidgin frontPage Story Promo I can see an image 1`] = `
<img
  alt="Senator Ahmad Lawan and Ali Ndume"
  class="StyledImg-sc-7vx2mr-0 hIxkbt"
  sizes="(max-width: 600px) 100vw, (max-width: 1008px) 50vw, 496px"
  src="https://ichef.bbci.co.uk/news/660/cpsprodpb/16D8E/production/_107328539_ninthnationalassemblypidgin_senate_reps_1080.jpg"
  srcset="https://ichef.bbci.co.uk/news/70/cpsprodpb/16D8E/production/_107328539_ninthnationalassemblypidgin_senate_reps_1080.jpg 70w, https://ichef.bbci.co.uk/news/95/cpsprodpb/16D8E/production/_107328539_ninthnationalassemblypidgin_senate_reps_1080.jpg 95w, https://ichef.bbci.co.uk/news/144/cpsprodpb/16D8E/production/_107328539_ninthnationalassemblypidgin_senate_reps_1080.jpg 144w, https://ichef.bbci.co.uk/news/183/cpsprodpb/16D8E/production/_107328539_ninthnationalassemblypidgin_senate_reps_1080.jpg 183w, https://ichef.bbci.co.uk/news/240/cpsprodpb/16D8E/production/_107328539_ninthnationalassemblypidgin_senate_reps_1080.jpg 240w, https://ichef.bbci.co.uk/news/320/cpsprodpb/16D8E/production/_107328539_ninthnationalassemblypidgin_senate_reps_1080.jpg 320w, https://ichef.bbci.co.uk/news/660/cpsprodpb/16D8E/production/_107328539_ninthnationalassemblypidgin_senate_reps_1080.jpg 660w"
  width="976"
/>
`;
